src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+// import libraries here
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    private static ArrayList<String> descriptions = new ArrayList<>();
+    private static ArrayList<Boolean> dones = new ArrayList<>();
+    private static ArrayList<String> types = new ArrayList<>();
+    private static ArrayList<String> dates = new ArrayList<>();
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greeting();
+        getCommand();
     }
-}
+
+    private static void getCommand() {
+        String line;
```

Comment:
The meaning of “line” is a bit unclear, maybe consider change it into "input"

Reply:
I will consider that.